engines/wazero/go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/wapc/wapc-go/engines/wazero
```

Comment:
One annoying thing I've found about multi module repos is that things like go tests and Golangci-lint don't automatically pickup all of the modules.

They require some magic tricks to properly catch things: https://github.com/tarmac-project/hord/blob/main/.github/workflows/lint.yml

Reply:
Surprising. Thanks for the heads up! Will investigate in a future PR.